src/integrations/CustomerIO/browser.js

```diff
@@ -30,10 +30,11 @@ class CustomerIO {
 
   identify(rudderElement) {
     logger.debug('in Customer IO identify');
-    const userId = rudderElement.message.userId
-      ? rudderElement.message.userId
-      : rudderElement.message.anonymousId;
-    const traits = rudderElement.message.context.traits ? rudderElement.message.context.traits : {};
+    const { userId, context } = rudderElement.message;
+    const { traits } = context || {};
+    if(!userId){
+      logger.error('userId is required for Identify call.');
```

Comment:
We don't need to return here ? 🤔

Reply:
Yes we do. Added now

Reply:
Will userId be only available in `message.userId` or should we consider something others as well (like how we do in [GenericFieldMapping.json](https://github.com/rudderlabs/rudder-transformer/blob/develop/src/v0/util/data/GenericFieldMapping.json#L12-L26))

Reply:
In most of the integrations we search for it at root level only but yes w can use GenericFieldMapping as well. Shall I make the change

Reply:
It is fine if it is being done for most of the integrations